package.json

```diff
@@ -12,7 +12,9 @@
     "lint": "nps lint",
     "test": "nps test",
     "build": "nps build",
-    "publish": "nps publish"
+    "publish": "nps publish",
+    "storybook": "start-storybook -p 6006",
```

Comment:
Move commands into `package-scripts.js`.
After that replace `demo` command with `storybook` command.